site/src/scaffold/styles/layout-buttons.scss

```
@@ -0,0 +1,10 @@
+$layouts: 'single', 'holy-grail', 'sidecar-left', 'sidecar-right';
+
+@each $layout in $layouts {
+	.layout-#{$layout} {
+		[data-set-layout='layout-#{$layout}'] {
+			@extend .button-primary;
+			@extend .button-filled;
+		}
+	}
+}
```

Bit of a simpler approach, given we apply `aria-pressed` and can use it as a [stateful, semantic selector](https://benmyers.dev/blog/semantic-selectors/)?

```suggestion
[data-set-layout][aria-pressed="true"] {
		@extend .button-primary;
		@extend .button-filled;
}
```

---

Looks good to me, a lot less output.